ee/frontend/mobile-replay/transformer/types.ts

```diff
@@ -0,0 +1,25 @@
+import {MobileStyles} from "../mobile.types";
+
+export interface ConversionResult<T> {
```

Comment:
TODO: reading through this now... sometimes I say transform and sometimes I saw convert but they're the same thing 🤦